CHANGELOG.md

```diff
@@ -78,6 +78,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - (ante) [#353](https://github.com/crypto-org-chain/ethermint/pull/353) Remove blocked address decorator and support custom decorators instead.
 - (statedb) [#359](https://github.com/crypto-org-chain/ethermint/pull/359) Add `CacheContext` method to StateDB, to support efficient read-only native actions.
 - (rpc) [#375](https://github.com/crypto-org-chain/ethermint/pull/375) Refactor websocket/subscription system to improve performance and stability.
+- (deps) [#381](https://github.com/crypto-org-chain/ethermint/pull/381) Upgrade Go-Ethereum version to [`v1.11.6`](https://github.com/ethereum/go-ethereum/releases/tag/v1.11.6).
```

Comment:
@yihuang need bump to [1.11.6](https://github.com/crypto-org-chain/go-ethereum/pull/14/files) first